### PR TITLE
Notification

### PR DESCRIPTION
**Pull request by hamza94max**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -5,12 +5,12 @@ android {
...
     defaultConfig {
-        applicationId "com.hazem.wazkar"
+        applicationId "com.hamza.wazkar"
         minSdkVersion 14
         //noinspection OldTargetApi,ExpiredTargetSdkVersion
         targetSdkVersion 30
-        versionCode 12
-        versionName "10.2"
+        versionCode 13
+        versionName "8.2"
```
Comment (MahmoudMabrok) [inline app/build.gradle:13]: should be 1 or at most 2 not 8 at all

File: app/src/main/java/com/hamza/Wazaker/Activites/MainActivity/MainActivity.java [hunk 2]
```diff
@@ -42,12 +50,19 @@ protected void onCreate(Bundle savedInstanceState) {
...
+    /**
+     * display notification every 8 hours
+     */
+    private void fireWorkerTodisplayNotification() {
+        WorkRequest WorkRequest =
+                new OneTimeWorkRequest.Builder(NotificationWorker.class)
+                        .setInitialDelay(8, TimeUnit.HOURS)
+                        .build();
+
+        workManager.enqueue(WorkRequest);
```
Comment (MahmoudMabrok) [inline app/src/main/java/com/hamza/Wazaker/Activites/MainActivity/MainActivity.java:65]: you should use Unique request so not to make new request each time app is opened

**Review by MahmoudMabrok — COMMENTED** [review]
check comments

File: app/build.gradle (hunk 1 above)
Comment (hamza94max) [reply app/build.gradle:13]: don't understand you 
It's just something for google play store

File: app/src/main/java/com/hamza/Wazaker/Activites/MainActivity/MainActivity.java (hunk 2 above)
Comment (hamza94max) [reply app/src/main/java/com/hamza/Wazaker/Activites/MainActivity/MainActivity.java:65]: it's right, Thank you for help and getting the project more cleaner

File: app/build.gradle (hunk 1 above)
Comment (MahmoudMabrok) [reply app/build.gradle:13]: you should reset your versioning of app 
if it X.Y.Z
so changing X for major changes 
Y for minor changes 
Z for bug fixes

**Pull request merged**